bgpd/bgp_route.c

```diff
@@ -583,6 +602,11 @@ static int bgp_path_info_cmp(struct bgp *bgp, struct bgp_path_info *new,
 	bool old_proxy;
 	bool new_proxy;
 	bool new_origin, exist_origin;
+	struct bgp_path_info *bpi_ultimate;
+
+	if (debug)
+		zlog_debug("%s: for bgp instance %s comparing path info",
+			   pfx_buf, bgp->name_pretty);
 
```

Comment:
this debug code should be removed. no need to add it

Reply:
We were thinking this log would be useful,
in debugging bestpath selection of imported routes.

We will remove this log.